### PR TITLE
Add MS Teams integration

### PR DESCRIPTION
**Pull request by jaffrepaul**
Add current basic steps to enable Microsoft Teams at both Org & Project levels.

### CODE REVIEW

**Comment by pstakoun** [comment]
The images are all pretty massive and make the text slightly hard to follow (e.g. select-project-settings.jpg takes up a lot of vertical space and would probably be just as effective if smaller).

File: docs/guides/cloud/integrations/ms-teams.mdx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+4. Click the **Enable** button in the Microsoft Teams section.
+
+   <DocsImage
+     src="/img/guides/cloud/integrations/enable-teams-integration.jpg"
+     alt="Click enable under Microsoft Teams Integration"
+     width={1000}
+   />
+
+5. You'll naviagte to a window that manages webhooks as a way for Cypress Cloud
```
Comment (pstakoun) [inline docs/guides/cloud/integrations/ms-teams.mdx:65]:
```suggestion
5. You'll navigate to a window that manages webhooks as a way for Cypress Cloud
```

File: docs/guides/cloud/integrations/ms-teams.mdx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+   to communicate with Microsoft Teams.
+
+   First, you must configure an Incoming Webhook to a Teams channel. For more
+   information on that, see
+   [Microsoft Teams - Create Incoming Webhooks](https://learn.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook?tabs=dotnet#create-incoming-webhooks-1).
+
+   Then, add the Teams webhook to your Cypress Cloud Organization, choosing the
+   types of notifications your team should recieve.
```
Comment (pstakoun) [inline docs/guides/cloud/integrations/ms-teams.mdx:73]:
```suggestion
   types of notifications your team should receive.
```

**Review by pstakoun — CHANGES_REQUESTED** [review]
Also note my comments on the PR regarding clear callouts of webhook creation via Teams and image sizes.

**Comment by jaffrepaul** [comment]
> The images are all pretty massive and make the text slightly hard to follow (e.g. select-project-settings.jpg takes up a lot of vertical space and would probably be just as effective if smaller).

These are reused assets from other pages (ie Slack integration) since they require the same steps. It depends on the viewport you're looking at them with but agree that some of these are sub-optimal. I'll edit some sizes and apply the changes to the other pages as well (separately) for now. There's ongoing work to optimize these Cloud pages across the board and some of these images may even be superfluous.

**Comment by jaffrepaul** [comment]
> I feel like creating Incoming Webhooks in Teams should be called out more "loudly" – maybe should have its own step

@pstakoun I added the steps via the MS page, similar to how we've done with GH integrations.

**Review by mabela416 — APPROVED** [review]
